test/infrastructure/vcsim/controllers/controlplaneendpoint_controller.go

```diff
@@ -132,7 +132,7 @@ func (r *ControlPlaneEndpointReconciler) SetupWithManager(ctx context.Context, m
 	err := ctrl.NewControllerManagedBy(mgr).
 		For(&vcsimv1.ControlPlaneEndpoint{}).
 		WithOptions(options).
-		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(ctrl.LoggerFrom(ctx), r.WatchFilterValue)).
+		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(mgr.GetScheme(), ctrl.LoggerFrom(ctx), r.WatchFilterValue)).
```

Comment:
I assume the change to pass in a different "predicateLog" is tracked somewhere and will be done in a separate PR?

(Just want to make sure we don't forget it)

Reply:
Updated in this PR to set it everywhere.